src/KawaBot.Server/Utils.fs

```diff
@@ -0,0 +1,10 @@
+﻿namespace KawaBot.Server
+
+open System.Collections.Generic
+
+[<AutoOpen>]
+module Utils =
+    let tryGet (dict: IReadOnlyDictionary<'K, 'V>) (key: 'K): option<'V> =
```

Comment:
consistency with function names in standard lib modules, like Option.map or Seq.unfold